tools/libboard/board_n6000/board_event_log.c

```diff
@@ -218,6 +219,27 @@ static void bel_print_header(const char *label, struct bel_header *header)
 	printf("  " BEL_LABEL_FMT "%s", 50, label, time_str);
 }
 
+static void reserved_field(void *p, uint32_t value, size_t first, size_t last)
+{
+	UNUSED_PARAM(p);  // Need dummy paramater to cheat the coding style check!
```

Comment:
Please don't do this.  The coding style check is there for a reason.  What is the issue?